### PR TITLE
AV8BNA migration to Module

### PR DESCRIPTION
**Pull request by jdahlblom**

### CODE REVIEW

**Review by charliefoxtwo — DISMISSED** [review]
Some minor points, but I'd really suggest we dedupe the `parse_indication` calls in a given function. In fact, it's even worth considering having a separate export hook that just handles the `parse_indication` and stores it to a local variable, and everything else reads off that. `parse_indication` is (relatively) expensive and so calling it as few times as possible will help with performance.

File: Scripts/DCS-BIOS/lib/modules/aircraft_modules/AV8BNA.lua [hunk 1]
```diff
@@ -0,0 +1,645 @@
...
+	return (" "):rep(2 - #txt) .. txt:sub(1, 2)
+end
+AV8BNA:defineString("UFC_COMM1_DISPLAY", getComm1Text, 2, "UFC Display", "UFC Comm1 Preset Display")
+
+local function getComm2Text()
+	if Module.parse_indication(5) == nil then
+		return (" "):rep(2)
+	end
+	local txt = Module.parse_indication(5)["ufc_chnl_2_m"] or ""
+	return (" "):rep(2 - #txt) .. txt:sub(1, 2)
```
[the remark this answers is not in the document]
Comment (jdahlblom) [reply Scripts/DCS-BIOS/lib/modules/aircraft_modules/AV8BNA.lua:566]: The last one I will leave. But I added a new export hook as suggested.

File: Scripts/DCS-BIOS/lib/modules/aircraft_modules/AV8BNA.lua [hunk 2]
```diff
@@ -0,0 +1,645 @@
...
+AV8BNA:defineBitFromDrawArgument("EXT_WOW_LEFT", 6, "External Aircraft Model", "Weight ON Wheels Left Gear")
+AV8BNA:defineBitFromDrawArgument("EXT_WOW_TAIL", 343, "External Aircraft Model", "Weight ON Wheels Tail Gear")
+
+-- Get Displays Functions
+local function getComm1Text()
+	if Module.parse_indication(5) == nil then
+		return (" "):rep(2)
+	end
+	local txt = Module.parse_indication(5)["ufc_chnl_1_m"] or ""
+	return (" "):rep(2 - #txt) .. txt:sub(1, 2)
```
[the remark this answers is not in the document]
Comment (jdahlblom) [reply Scripts/DCS-BIOS/lib/modules/aircraft_modules/AV8BNA.lua:557]: I can even decode that one, I rather leave it.

File: Scripts/DCS-BIOS/lib/modules/aircraft_modules/AV8BNA.lua [hunk 3]
```diff
@@ -0,0 +1,645 @@
...
+end
+AV8BNA:defineString("UFC_COMM1_DISPLAY", getComm1Text, 2, "UFC Display", "UFC Comm1 Preset Display")
+
+local function getComm2Text()
+	if Module.parse_indication(5) == nil then
```
[the remark this answers is not in the document]
Comment (jdahlblom) [reply Scripts/DCS-BIOS/lib/modules/aircraft_modules/AV8BNA.lua:562]: done

File: Scripts/DCS-BIOS/lib/modules/aircraft_modules/AV8BNA.lua [hunk 4]
```diff
@@ -0,0 +1,645 @@
...
+AV8BNA:defineFloatFromDrawArgument("EXT_FORMATION_LIGHTS", 201, "External Aircraft Model", "Formation Lights (green)")
+AV8BNA:defineBitFromDrawArgument("EXT_STROBE", 83, "External Aircraft Model", "Strobe Lights (red)")
+AV8BNA:defineBitFromDrawArgument("EXT_WOW_NOSE", 1, "External Aircraft Model", "Weight ON Wheels Nose Gear")
+AV8BNA:defineBitFromDrawArgument("EXT_WOW_RIGHT", 4, "External Aircraft Model", "Weight ON Wheels Right Gear")
+AV8BNA:defineBitFromDrawArgument("EXT_WOW_LEFT", 6, "External Aircraft Model", "Weight ON Wheels Left Gear")
+AV8BNA:defineBitFromDrawArgument("EXT_WOW_TAIL", 343, "External Aircraft Model", "Weight ON Wheels Tail Gear")
+
+-- Get Displays Functions
+local function getComm1Text()
+	if Module.parse_indication(5) == nil then
```
[the remark this answers is not in the document]
Comment (jdahlblom) [reply Scripts/DCS-BIOS/lib/modules/aircraft_modules/AV8BNA.lua:553]: done

**Pull request merged**